content/guides/local-quickstart.md

```diff
@@ -353,7 +353,7 @@ git clone -b main https://github.com/deployKF/deployKF.git ./deploykf
 chmod +x ./deploykf/argocd-plugin/install_argocd.sh
 
 # run the install script
-# WARNING: this will install into your current kubectl context
+# WARNING: this will install into ARGOCD_NAMESPACE (default "argocd")
```

Comment:
We should still tell users that it will install into their current kubectl context.

Perhaps we could make it:

```
# WARNING: this will install into your current kubectl context (namespace: "argocd")
```

Reply:
I might be mistaking but that suggestion is still incorrect. Nowhere in the quickstart do we mention seting ARGOCD_NAMESPACE or modifying the script that actual runs (where it also defaults to argocd), and the current-context (in k8s parlance) is not actual argocd; thus always installs in argocd (per quickstart, doesn't mean we can't tell them to update it).

The comment in the doc, right after installing is correct though.

> After the script completes, wait for all pods in the argocd Namespace to be in a Running state ([using k9s or kubectl as described above](https://www.deploykf.org/guides/local-quickstart/#wait-for-cluster-to-be-ready)).